scripts/globals/garrison.lua

```diff
@@ -485,16 +535,26 @@ local spawnNPC = function(zone, x, y, z, rot, name, look)
     return mob
 end
 
+-- Spawns all npcs for the zone in the given garrison starting npc
 local spawnNPCs = function(npc)
     local zone     = npc:getZone()
     local zoneData = xi.garrison.data[zone:getID()]
     local xPos     = zoneData.xPos
     local yPos     = zoneData.yPos
     local zPos     = zoneData.zPos
     local rot      = zoneData.rot
+
+    -- There's a bug on first time zone load where these allies array are null for some reason.
+    -- Since they are all the same for all zones, this is fine but we should really
+    -- find out why that's happening.
+    if zoneData.allies == nil then
```

Comment:
this was bad, and it really puzzled me because of what the symptoms are.
Every time I restart the zone / xi_map, zoneData is correct, but for some reason the allies table is nil.